static/js/compose.js

```diff
@@ -666,7 +666,11 @@ exports.initialize = function () {
         var video_call_id = util.random_int(100000000000000, 999999999999999);
         var video_call_link = 'https://meet.jit.si/' +  video_call_id;
         var video_call_link_text = '[' + _('Click to join video call') + '](' + video_call_link + ')';
-        compose_ui.insert_syntax_and_focus(video_call_link_text);
+        video_call_link_text = compose_ui.insert_syntax_and_focus(video_call_link_text);
+        if (!document.execCommand("insertText", false, video_call_link_text)) {
+            var textarea = $("#new_message_content");
+            textarea.caret(video_call_link_text);
+        }
```

Comment:
I think rather than doing this in each place, we'd rather have `insert_syntax_and_focus` do the `document.execCommand` stuff; that way, we only need this logic to appear once in the codebase...

Reply:
Agree!!